### PR TITLE
VEIOSS-470 - PaymentRequest to be updated

### PR DESCRIPTION
**Pull request by modivilioglu**

### CODE REVIEW

File: test/connectors/FinancialDataConnectorSpec.scala [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  private val now = zonedNow.toLocalDate
+  "getFinancialData" - {
+    def financialDataUrl(commencementDate: LocalDate) = s"/ioss-returns/financial-data/get/$commencementDate"
+
+    "when the server returns OK and a recognised payload" - {
+      "must return a FinancialDataResponse" in {
+        val app = application
+
+        server.stubFor(
+          get(urlEqualTo(financialDataUrl(now)))
```
Comment (hmrcvmu) [inline test/connectors/FinancialDataConnectorSpec.scala:48]: I think you could inline the call `get(urlEqualTo(s"/ioss-returns/financial-data/get/$now"))`

File: test/connectors/FinancialDataConnectorSpec.scala [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+  "prepareData" - {
+    def prepareDataUrl() = s"/ioss-returns/financial-data/prepare"
+
+    "when the server returns OK and a recognised payload" - {
+      "must return a FinancialDataResponse" in {
+        val app = application
+
+        server.stubFor(
+          get(urlEqualTo(prepareDataUrl()))
```
Comment (hmrcvmu) [inline test/connectors/FinancialDataConnectorSpec.scala:70]: Inline? `get(urlEqualTo("/ioss-returns/financial-data/prepare"))`

File: test/connectors/FinancialDataConnectorSpec.scala [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+      |  "iossNumber": "IM9001234567"
+      |
+      |}""".stripMargin
+
+  val responseJsonFinancialData: String =
+    s"""{
+       | "idType": "IOSS",
+       | "idNumber": "123456789",
+       | "regimeType": "ECOM",
+       | "processingDate": "${zonedNow.toString}",
```
Comment (hmrcvmu) [inline test/connectors/FinancialDataConnectorSpec.scala:137]: `$zonedNow`

File: test/connectors/FinancialDataConnectorSpec.scala (hunk 1 above)
Comment (modivilioglu) [reply test/connectors/FinancialDataConnectorSpec.scala:48]: You're right, as it's used only once. Done.

File: test/connectors/FinancialDataConnectorSpec.scala (hunk 2 above)
Comment (modivilioglu) [reply test/connectors/FinancialDataConnectorSpec.scala:70]: You're right, as it's used only once. Done.

File: test/connectors/FinancialDataConnectorSpec.scala (hunk 3 above)
Comment (modivilioglu) [reply test/connectors/FinancialDataConnectorSpec.scala:137]: Done.

**Review by hmrcvmu — APPROVED** [review]

**Pull request merged**